plaid/src/apis/npm/mod.rs

```diff
@@ -22,11 +25,18 @@ impl Npm {
         let client = Client::builder()
             .cookie_provider(cookie_jar.clone())
             .build()
-            .map_err(|_| NpmError::GenericError).unwrap(); // TODO @obelisk OK to unwrap here? If we cannot build the client, perhaps we should just panic?
+            .map_err(|_| NpmError::GenericError)
+            .unwrap(); // TODO @obelisk OK to unwrap here? If we cannot build the client, perhaps we should just panic?
```

Comment:
This comes with a philosophical question about Plaid: if an API is misconfigured or cannot start, what should be done. Based on the fact that you have `NpmError::GenericError` we *could* handle it. My preference would be to make the error more descriptive and then return control back to the higher level to decide if API configuration failure are something it wants to allow (that means in the future we can make the decision in one place instead of in every module) but I don't feel so strongly about it as to block the PR on it.

Reply:
Discussed offline and solved: for now, an error here is propagated to `Api::new`, we log an error, and Plaid's initialization continues